src/main/webapp/app/modules/canvas/features/modals/create/features-create-modal.tsx

```diff
@@ -46,6 +70,16 @@ const FeaturesCreateModal = (props: IFeaturesCreateModalProps) => {
     return false;
   };
 
+  if (props.succeeded || canceled) {
```

Comment:
here also cancelled

Reply:
are we preferring British english for naming variables and methods?